include/mlir/IR/OpImplementation.h

```diff
@@ -567,7 +567,8 @@ class OpAsmParser {
   virtual ParseResult parseColonType(Type &result) = 0;
 
   /// Parse a colon followed by a type of a specific kind, e.g. a FunctionType.
-  template <typename TypeType> ParseResult parseColonType(TypeType &result) {
+  template <typename TypeType>
+  ParseResult parseColonType(TypeType &result) {
```

Comment:
This seems like an unrelated format change, likely caused by a different version of clang-format.

Reply:
Which version of clang-format should I be using?

Reply:
I’m happy to fix this by using the same version as you. Could you share your version?

Reply:
This is caused by:
https://github.com/tensorflow/mlir/blob/master/.clang-format#L2